src/main/java/edu/hm/hafner/analysis/parser/EclipseParser.java

```diff
@@ -80,20 +79,20 @@ static void extractMessage(final IssueBuilder builder, final String message) {
     }
 
     /**
-     * Sets the issue's category to {@code Code} or {@code Javadoc}. Unlike
-     * {@link #extractMessage(IssueBuilder, String)}, the {@code message} is assumed to be cleaned-up.
+     * Sets the issue's category to {@code Javadoc} if the message starts with {@value #CAT_PREFIX}, empty string
+     * otherwise. Unlike {@link #extractMessage(IssueBuilder, String)}, the {@code message} is assumed to be cleaned-up.
      * 
      * @param builder
      *     IssueBuilder to populate.
      * @param message
      *     issue to examine.
      */
     static void extractCatagory(final IssueBuilder builder, final String message) {
-        if (message == null || !message.startsWith(CAT_PREFIX)) {
-            builder.setCategory(CAT_CODE);
+        if (message != null && message.startsWith(JAVADOC_PREFIX)) {
+            builder.setCategory(Categories.JAVADOC);
         }
         else {
-            builder.setCategory(CAT_JAVADOC);
+            builder.setCategory("");
```

Comment:
I'll use `Other` here, and `Other` and `Unknown` for the XML version. (There is a case `"0"` I omitted since it's the internal Eclipse unknown reason, and other for the default case)

Reply:
Scratch that about XML, forgot that it calls the `"0"` case 'Unspecified'